_pages/install/schematics.md

```diff
@@ -67,7 +67,7 @@ When you run `ng add @spartacus/schematics`, the command does the following:
    - Adds server-side rendering dependencies.
    - Provides additional files that are required for SSR.
 
-### CMS Component Schematic
+## CMS Component Schematic
```

Comment:
Nice, this one should be on the same level as `Adding Spartacus Core Libraries and Features to Your Angular Project` and `Installing Additional Spartacus Libraries` 👍

Reply:
😅